package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "mvom",
   "private": true,
-  "version": "2.0.0-rc.1",
+  "version": "2.0.0",
```

Comment:
Don't change the version until a new release is being prepared.

Reply:
Fixed.